lua/starfall/libs_sv/entities.lua

```diff
@@ -775,3 +775,34 @@ function ents_methods:setUnbreakable(on)
 	ent:Fire( "SetDamageFilter", on and "FilterDamage" or "", 0 )
 end
 
+--- Check if the given Vector is within this entity's PVS (Potentially Visible Set). See: https://developer.valvesoftware.com/wiki/PVS
+-- @param pos Vector positon to test
+-- @return bool True/False
+function ents_methods:testVectorPVS (pos)
+	checktype(self, ents_metatable)
+	checktype(pos, SF.Types["Vector"])
+	
+	local this = unwrap(self)
+	local pos = vunwrap(pos)
+
+	if not this and not this:IsValid() then SF.Throw("Entity is not valid", 2) end
```

Comment:
Should be `if not this or not this:IsValid() then` or `if not (this and this:IsValid()) then`